addons/gamememo/src/presenter.js

```diff
@@ -1647,14 +1674,27 @@ function Addongamememo_create(){
         presenter.isShowErrorsMode = false;
     };
 
-    presenter.onEventReceived = function (eventName) {
+    presenter.onEventReceived = function (eventName, data) {
         if (eventName == "ShowAnswers") {
             presenter.showAnswers();
         }
 
         if (eventName == "HideAnswers") {
             presenter.hideAnswers();
         }
+
+        if (eventName == "GradualShowAnswers") {
+            if (presenter.configuration.ID == data.moduleID) {
+                presenter.gradualShowAnswers();

```

Comment:
Gdy GradualShowAnswer wchodzi do addonu to nie ukrywa aktualnych odpowiedzi użytkownika. 
W momencie wejścia do addonu odkryte karty memo powinny się schować i następnie po kolei powinny pojawiać się poprawne odpowiedzi zgodnie z komentarzem:
https://learneticsa.assembla.com/spaces/lorepo/tickets/7899/details?comment=1692478290

Przykładowa lekcja gdzie można sprawdzić tą sytuację:
https://test-8073-dot-mauthor-dev.ew.r.appspot.com/embed/4659481248006144#8
Rekontrukcja
1. Udzielić kilka odpowiedzi w memo game
2. Zacząć Klikanie GSA button

Reply:
Na pewno ma to działać w ten sam sposób w przypadku tego addonu? W sensie poświęciłem trochę czasu, żeby to działało w ten sposób, bo wydawało mi się to najbardziej logiczne - w tym addonie jeśli użytkownik znalazł 2 odpowiedzi, to znaczy, że one na 100% będą poprawne (w przeciwieństwie do innych addonów, gdzie gdy użytkownik wprowadza odpowiedź, to może ona być zarówno dobra jak i zła - nie wiadomo tego aż do czasu sprawdzenia odpowiedzi)